src/commands/codepush/lib/cordova-utils.ts

```diff
@@ -0,0 +1,49 @@
+import * as which from "which";
+import * as xml2js from "xml2js";
+import * as fs from "fs";
+import * as path from "path";
+
+export function getCordovaProjectAppVersion(): Promise<string> {
+  return new Promise<string>((resolve, reject) => {
+    try {
+      const projectRoot: string = process.cwd();
```

Comment:
Seems odd to hard-code this to the current directory, don't you want to give users the option of where to get this from?

Reply:
The `codepush release-react/cordova` commands are designed to be executed in the root dirs of the projects we are wanting to release updates.
But Agreed, we can update the method signature to make it possible to specify the `projectRoot`. We'll prepare another one PR with several fixes/improvements for release commands.